kernel/build-kernel.sh

```diff
@@ -90,6 +92,7 @@ Commands:
 Options:
 
 	-c <path>   : Path to config file to build a the kernel.
+	-b          : Enable BPF related features in kernel.
```

Comment:
Same comment as above. Not necessary for this PR, but I'd personally remove the hard-coded output of the script in the README and tell users they can run `build-kernel.sh -h` for further info in the doc - easier maintenance! :smile: